triager/config.py

```diff
@@ -10,70 +11,117 @@
 class Config:
     def __init__(self, cfg):
         # select config.yaml from cwd
+        logging.debug(f"Config.__init__ called with cfg: {cfg}")
         if not cfg:
             logging.info("config file not specified, setting default")
             cfg = "./config.yaml"
 
-        logging.info("attempting to read config file: {0}".format(cfg))
+        logging.info(f"attempting to read config file: {cfg}")
 
         try:
             with open(cfg, "r") as config_file:
-                config = yaml.safe_load(config_file)
-            logging.info("config file successfully loaded")
+                config_content = config_file.read()
```

Comment:
Normalize these class operations and break them into smaller methods.

Reply:
done updated in recent commit